### PR TITLE
Rework Debugging Info output

### PR DESCRIPTION
**Pull request by underyx**
PR checklist:

- [ ] Purpose of the code is [evident to future readers](https://semgrep.dev/docs/contributing/contributing-code/#explaining-code)
- [ ] Tests included or PR comment includes a reproducible test plan
- [ ] Documentation is up-to-date
- [ ] A changelog entry was [added to changelog.d](https://semgrep.dev/docs/contributing/contributing-code/#adding-a-changelog-entry) for any user-facing change
- [ ] Change has no security implications (otherwise, ping security team)

If you're unsure on any of this, please see:

- [Contribution guidelines](https://semgrep.dev/docs/contributing/contributing-code)!
- [One of the more specific guides located here](https://semgrep.dev/docs/contributing/contributing/)

### CODE REVIEW

File: cli/src/semgrep/commands/ci.py [hunk 1]
```diff
@@ -308,8 +327,11 @@ def ci(
     if dataflow_traces is None:
         dataflow_traces = engine_type.has_dataflow_traces
 
-    console.print(Title("Scan Environment", order=2))
-    console.print(Padding(scan_env, (0, 2)))
+    if max_memory is None:
+        max_memory = engine_type.default_max_memory
+
+    if interfile_timeout is None:
+        interfile_timeout = engine_type.default_interfile_timeout
```
Comment (brendongo) [inline cli/src/semgrep/commands/ci.py]: didn't you fix this in a different way in the other PR

File: cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt [hunk 2]
```diff
@@ -63,13 +63,15 @@ First-Party Blocking Rules Fired:
...
   SCAN ENVIRONMENT
-    versions    - semgrep <MASKED> on python <MASKED>
-    environment - running in environment git, triggering event is unknown  
-    server      - logged in as org_name on https://semgrep.dev             
-               
+  versions    - semgrep <MASKED> on python <MASKED>
+  environment - running in environment git, triggering event is unknown
+            
+  CONNECTION
+Would have sent POST request to create scan
```
Comment (brendongo) [inline cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt:72]: This indent seems wrong?

**Review by brendongo — APPROVED** [review]

File: cli/src/semgrep/commands/ci.py (hunk 1 above)
Comment (underyx) [reply cli/src/semgrep/commands/ci.py]: lol

CleanShot 2023-03-02 at 00 07 50@2x

File: cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt (hunk 2 above)
Comment (underyx) [reply cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt:72]: @brendongo yeah, cause that line is not within rich yet. The design doc asks that we prefix lines like this with

```
! An arbitrary non-fatal error occurred. Here is the warning message.
```

but that's not done yet

**Pull request merged**